garak/analyze/report_digest.py

```diff
@@ -340,5 +514,17 @@ def compile_digest(
     taxonomy = None
     if len(sys.argv) == 3:
         taxonomy = sys.argv[2]
-    digest_content = compile_digest(report_path, taxonomy=taxonomy)
+
+    digest = _get_report_digest(report_path)
+    if not digest:
+        digest = build_digest(report_path)
+        if _config.reporting.write_digests_to_existing_reports:
```

Comment:
Do not access `_config` when invoking this file as a script.  The flag tested here should be passed on the in `sys.argv`.